bw2data/project.py

```diff
@@ -1,18 +1,18 @@
 import os
 import shutil
-import tempfile
 import warnings
 from collections.abc import Iterable
 from pathlib import Path
 from typing import Optional
 
-import wrapt
 from bw_processing import safe_filename
 from peewee import SQL, BooleanField, DoesNotExist, Model, TextField
 from platformdirs import PlatformDirs
+import wrapt
 
 from . import config
 from .filesystem import create_dir
+from .signals import project_changed, project_created
```

Comment:
Slightly unrelated but imo we should really aim to transition into using [absolute imports][imports].

[imports]: https://peps.python.org/pep-0008/#imports

Reply:
https://github.com/brightway-lca/brightway2-data/pull/185